src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendRecorder.java

```diff
@@ -166,6 +164,7 @@ private TraceFilterForm buildFilterFormForCondition(final ThresholdCondition con
 
     @Override
     public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, final BuildListener listener) throws IOException, InterruptedException {
+        final String SKIP_APPLICATION_MESSAGE = "Error encountered. Skipping application.";
 
```

Comment:
Is this a UX decided phrasing. In combination with "Application with [name = %s, agentType = %s] not found." We could almost just have "Skipping application" since I think the former provides enough context.

Reply:
updated